bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -90,31 +87,6 @@ public ItemChannelLink update(ItemChannelLink element) {
         return linkedItems;
     }
 
-    /**
-     * Returns a set of bound things for the given item name.
-     *
-     * @deprecated Will be removed soon. Use {@link ItemThingLinkRegistry#getLinkedThings(String)} instead.
-     *
-     * @param itemName
-     *            item name
-     * @return set of bound things for the given item name
-     */
-    @Deprecated
```

Comment:
I think this is an error. ItemThingLinkRegistry#getLinkedThings(String) is not a replacement for this method. ItemThingLinkRegistry#getLinkedThings(String) returns a thing, that is linked to an item directly, like the old items that were previously created for each thing. But this method here returns a collection of things, that are connected via channels to this specific item.

For example, getBoundThings("hue_0210_001788141f1a_1_color") would return the thing UID ["hue:0210:001788141f1a:1"], but ItemThingLinkRegistry#getLinkedThings("hue_0210_001788141f1a_1_color") would return []. 

Only ItemThingLinkRegistry#getLinkedThings("hue_0210_001788141f1a_1") would return ["hue:0210:001788141f1a:1"] (requesting using the old ThingItem).